ext/taglib_id3v2/taglib_id3v2_wrap.cxx

```diff
@@ -11269,7 +11422,10 @@ SWIGEXPORT void Init_taglib_id3v2(void) {
   rb_define_const(SwigClassAttachedPictureFrame.klass, "Illustration", SWIG_From_int(static_cast< int >(TagLib::ID3v2::AttachedPictureFrame::Illustration)));
   rb_define_const(SwigClassAttachedPictureFrame.klass, "BandLogo", SWIG_From_int(static_cast< int >(TagLib::ID3v2::AttachedPictureFrame::BandLogo)));
   rb_define_const(SwigClassAttachedPictureFrame.klass, "PublisherLogo", SWIG_From_int(static_cast< int >(TagLib::ID3v2::AttachedPictureFrame::PublisherLogo)));
+  rb_define_singleton_method(SwigClassAttachedPictureFrame.klass, "type_to_string", VALUEFUNC(_wrap_AttachedPictureFrame_type_to_string), -1);
```

Comment:
Same question as with FLAC::Picture. We now have both `TagLib::ID3v2::AttachedPictureFrame.type_to_string` and `TagLib::ID3v2.picture_type_to_string`.